core/src/cf/mutations.rs

```diff
@@ -250,7 +281,7 @@ mod tests {
 		let s = serde_json::to_string(&v).unwrap();
 		assert_eq!(
 			s,
-			r#"{"changes":[{"create":{"id":"mytb:tobie","note":"surreal"}},{"update":{"id":"mytb:tobie2","note":"surreal"}},{"delete":{"id":"mytb:tobie"}},{"define_table":{"name":"mytb"}}],"versionstamp":1}"#
+			r#"{"changes":[{"original":{"id":"mytb:tobie","note":"surreal"},"update":[{"op":"add","path":"/`/temp`","value":true}]},{"original":{"id":"mytb:tobie2","note":"surreal"},"update":[{"op":"remove","path":"/`/temp`"}]},{"delete":{"id":"mytb:tobie"}},{"define_table":{"name":"mytb"}}],"versionstamp":1}"#
```

Comment:
I understand this is fake data, but I would love to have a test that represents a real use case.

If this is the case, then I don't understand how the value of `update` relates to `original`. When a user receives this diff, what kind of knowledge should they infer from it?

A straightforward diff for me would be:
```
{
  "changes": [
    {
      "original": {
        "id": "mytb:tobie",
        "name": "Tobie"
      },
      "update": [
        {
          "op": "add",
          "path": "/company",
          "value": "surrealdb"
        }
      ]
    }
  ]
}
```

Given this diff, I can easily see that the resulting value will be:
```
{
  "id": "mytb:tobie",
  "name": "Tobie",
  "company": "surrealdb"
}
```

Reply:
Yes, exactly. I will actually change that part of the test so that its object comparison instead of string. It was really unpredictable in terms of fixing it.

Reply:
Maybe I don't understand what these diffs mean.

For example, looking at this:
`{"original":{"id":"mytb:tobie2","note":"surreal"},"update":[{"op":"remove","path":"/`/temp`"}]}`

How can an `update` operation `remove` `/temp` if it doesn't exist in the original object? 🤔

Reply:
I still think the concept of a DIFF notification is still not clear, but won't block the PR 👍

Reply:
I have solved this with https://github.com/surrealdb/surrealdb/pull/3552/commits/b4f83b0af04e336325b7f06ff63079c27bbb25c6
Will DM and attach to linear SUR-339